packages/components/src/components/custom-select/custom-select.lite.tsx

```diff
@@ -239,8 +242,9 @@ export default function DBCustomSelect(props: DBCustomSelectProps) {
 				}`;
 			}
 		},
-		handleTagRemove: (option: CustomSelectOptionType, event: any) => {
-			event.stopPropagation();
+		handleTagRemove: (option: CustomSelectOptionType, event?: any) => {
```

Comment:
Could we use MouseEvent instead of any?
```suggestion
		handleTagRemove: (option: CustomSelectOptionType, event?: MouseEvent) => {
```